deb-package/pkg/etc/dedis/dvoting/config.env

```diff
@@ -0,0 +1,12 @@
+# dela config
+export dela_bin="/opt/dedis/dvoting/bin/memcoin"
+export dela_data="/var/opt/dedis/dvoting/data/dela"
+export dela_port="9000"
+export prometheus_addr="127.0.0.1:9100"
+export proxy_addr="127.0.0.1:9080"
```

Comment:
nothing in this file seems to belong to `/etc`, except these 3 lines.
Keep in mind, `/etc` files do not get automatically updated when updating the package, as they're automatically marked as config files and considered user-modifiable.